src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al

```diff
@@ -20,7 +20,7 @@ page 7775 "Copilot AI Capabilities"
     DataCaptionExpression = '';
     AboutTitle = 'About Copilot';
     AboutText = 'Copilot is the AI-powered assistant that helps people across your organization unlock their creativity and automate tedious tasks.';
-    AdditionalSearchTerms = 'OpenAI,AI,Copilot,Co-pilot,Artificial Intelligence,GPT,GTP,Dynamics 365 Copilot,ChatGPT,Copilot settings,Copilot setup,enable Copilot,Copilot admin';
+    AdditionalSearchTerms = 'OpenAI,AI,Copilot,Co-pilot,Artificial Intelligence,GPT,GTP,Dynamics 365 Copilot,ChatGPT,Copilot settings,Copilot setup,enable Copilot,Copilot admin,Copilot and';
```

Comment:
I assume the "Copilot and" is an intentional addition? Haven't seen that before, but I guess it makes sense. The "AI capabilities" part isn't needed when users should choose to type and rather than &.